client/client_test.go

```diff
@@ -81,6 +82,36 @@ func testExtraHosts(t *testing.T, sb integration.Sandbox) {
 	require.NoError(t, err)
 }
 
+func testNetworkMode(t *testing.T, sb integration.Sandbox) {
+	t.Parallel()
+
+	c, err := New(context.TODO(), sb.Address())
+	require.NoError(t, err)
+	defer c.Close()
+
+	st := llb.Image("busybox:latest").
+		Run(llb.Shlex(`sh -c 'ping -c 1 8.8.8.8 2>&1 | grep "Network is unreachable"'`), llb.Network(llb.NetModeNone))
```

Comment:
`wget https://example.com`? (ping does not work for unprivileged users by default)